### PR TITLE
UICIRC-601: Add RTL/Jest testing for ScedulesList component

### PR DESCRIPTION
**Pull request by Stanislau-Snapok**
## Purpose
Add RTL/Jest tests for component ScedulesList (DetailsSection)

## Approach
Function with tests was extracted for posibility run tests separatly for each part of incoming data

## Refs
https://issues.folio.org/browse/UICIRC-601

## Screenshots
image

### CODE REVIEW

File: test/jest/__mock__/stripesComponents.mock.js [hunk 1]
```diff
@@ -28,4 +28,6 @@ jest.mock('@folio/stripes/components', () => ({
     <div ref={ref} {...rest}>{children}</div>
   )),
   PaneMenu: jest.fn((props) => <div>{props.children}</div>),
+  Row: jest.fn(({ children, ...rest }) => <div {...rest}>{children}</div>),
+  Col: jest.fn((props) => <div>{props.children}</div>),
```
Comment (annamelnyk) [inline test/jest/__mock__/stripesComponents.mock.js:32]: Lets keep alphabetical order of stripes components here

**Comment by annamelnyk** [comment]
Great work, @Stanislau-Snapok!

File: test/jest/__mock__/stripesComponents.mock.js (hunk 1 above)
Comment (Stanislau-Snapok) [reply test/jest/__mock__/stripesComponents.mock.js]: refactored

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,7 @@
 * Update `codemirror` and `react-codemirror2`. Refs UICIRC-576.
 * Update `folio/stripes-template-editor` to `3.0.0`. Refs UICIRC-575
 * Fix bug causing incorrect validation in lost item fee policies. Fixes UICIRC-667, UICIRC-668.
+* Add RTL/Jest testing for ScedulesList component
```
[the remark this answers is not in the document]
Comment (Stanislau-Snapok) [reply CHANGELOG.md]: refactored

**Review by annamelnyk — APPROVED** [review]

**Review by Dmitriy-Litvinenko — APPROVED** [review]

**Pull request merged**